db/psql/getAllClients.js

```diff
@@ -0,0 +1,17 @@
+var db = require('../config.js');
+//This function gets all clients from salesperson_client for a specific salesperson
+
+
+var getAllClientsString = "SELECT * FROM client INNER JOIN salesperson_client on (salesperson_client.client_id = client.client_id) WHERE salesperson_client.salesperson_id = $1"
+
+module.exports = function(salesPersonId){
```

Comment:
done! it's promisifed so you only get the results. so invocation is callback(result)